sdk/trace/span.go

```diff
@@ -189,15 +189,18 @@ func (s *recordingSpan) SetStatus(code codes.Code, description string) {
 	if !s.IsRecording() {
 		return
 	}
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	if s.status.Code == codes.Ok {
```

Comment:
If the status is Error, it should also be impossible to set it to Unset. So how about:

```suggestion
	if s.status.Code >= status.Code {
```

Reply:
Good point.
This particular implementation would only record the first call to SetStatus in case of error, while I believe the spec says that the last call should get priority.

```suggestion
	if s.status.Code > status.Code {
```

would work, but be slightly wasteful for multiple calls with `Ok`. But I suppose that's rare enough to not matter.